narwhals/series.py

```diff
@@ -519,6 +519,40 @@ def mean(self) -> Any:
         """
         return self._compliant_series.mean()
 
+    def skew(self) -> Any:
```

Comment:
Same as `Expr.skew`, polars exposes a `bias` parameter

Reply:
I'm not sure how to expose this `bias` parameter in Narwhals since its only available in Polars, not in Pandas and Arrow. Do you have ideas?

Reply:
See conversation in `narwhals/expr.py`